tcms/testruns/templates/testruns/get.html

```diff
@@ -311,16 +312,26 @@ <h5><span class="test-executions-count"></span> Results</h5>
 
                                             </div>
                                         </div>
-                                        <div class="col-md-3">
-                                            <span>{% trans 'Test Case Attachments' %}:</span>
-                                            <ul class="list-group list-unstyled test-case-attachments">
-                                                <template id="attachments-list-item">
-                                                    <li class="list-group-item'">
-                                                        <a></a>
-                                                    </li>
-                                                </template>
-                                                <li class="list-group-item hidden">{% trans "No attachments" %}</li>
-                                            </ul>
+                                        <div class="col-md-6">
+                                            <div>
+                                                <h4> {% trans "Comments "%} </h4>
+                                                {{ comment_form.text }}
+                                                <div class="form-group" style="padding: 15px; text-align: right;">
+                                                    <button class="btn post-comment">{% trans "Save" %}</button>
```

Comment:
Needs "btn-default" instead of the previous "btn-primary" class, otherwise looks very different from everything else.